Sources/HStack.swift

```diff
@@ -4,15 +4,15 @@
 import UIKit
 
 open class HStack: Stack {
-    public let thingsToStack: [Stackable]
+    public let thingsToStack: [StackableProtocol]
```

Comment:
Will this go out with an version bump?

Reply:
yep! it'll get merged into `release/2.0.0` for now, and when that is merged into master it will be a major bump. including a PR that is coming

Reply:
O totally missed that, cool 👍